src/ember-resource.js

```diff
@@ -1289,11 +1316,21 @@
       return length;
     }),
 
-    content: Ember.computed(function(name, value) {
-      if (arguments.length === 2) { // setter
-        return this.instantiateItems(value);
-      }
-    }),
+    content: requiresEmberComputedPropertyFunction ?
+      Ember.computed(function(name, value) {
+        if (arguments.length === 2) { // setter
+          return this.instantiateItems(value);
+        }
+      }) :
+      Ember.computed( {
+        set: function(name, value) {
+          return this.instantiateItems(value);
+        },
+
+        get: function() {
+          return;
+        }
+      }),
```

Comment:
correct, this was already the case in the original code, just making sure the changes are not introducing any side effect.